usl_pipeline/study_area_uploader/study_area_uploader/main.py

```diff
@@ -153,5 +158,12 @@ def parse_args() -> argparse.Namespace:
         help="Band index in GeoTIFF file containing elevation data (default value is"
         + " 1)",
     )
+    parser.add_argument(
+        "--verbose",
+        type=bool,
```

Comment:
I think you'd need to do something like
```python
parser.add_argument('--verbose', action=argparse.BooleanOptionalAction)
```
As is, I think it'd treat this like `--verbose argument_you_passed_into_verbose_arg`

Reply:
Yep! Done.